include/oneapi/dpl/pstl/parallel_backend_tbb.h

```diff
@@ -906,7 +912,8 @@ class __merge_func
         else
         {
             __move_range()(_M_z_beg + _M_zs, _M_z_beg + _M_zs + __nx, _M_x_beg + _M_xs);
-            __cleanup_range()(_M_z_beg + _M_zs, _M_z_beg + _M_zs + __nx);
+            __cleanup_range()(_M_z_beg + _M_zs, _M_z_beg + _M_zs + __nx,
```

Comment:
Why is a difference approach used here?
(see the comment above)

Reply:
Fixed